pkg/chartutil/testdata/test-values-with-extra.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "addresses": {
+      "$ref": "https://helm.sh/schemas/addresses"
```

Comment:
Does the change add support for referencing JSON Schemas from local files as well? For instance:

```json
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "properties": {
    "addresses": {
      "$ref": "file://./test-values-extra.schema.json"
```

Reply:
What about `$ref`-loading a schema that sits in a parent directory? For example: using the schema of a chart that sits in the same repo?

```
🐙 repo
  📁 charts
    📁 apps
  ┌─► 📦 app-1
  │     📜 Chart.yaml
  │     📄 schema.json ◄──────────────────────────────────────────────┐
  ┌─► 📦 app-2                                                        │
  │     📜 Chart.yaml                                                 │
  │     📄 schema.json ◄──────────────────────────────────────────────┐
  │   📦 app-3                                                        │
  │     📜 Chart.yaml                                                 │
  │       > dependencies:                                             │
  │       >   - name: app-1                                           │
  │       >     version: 0.0.0                                        │
  └────── >     repository: file://../app-1                           │
  │       >   - name: app-2                                           │
  │       >     version: 0.0.0                                        │
  └────── >     repository: file://../app-2                           │
        📄 schema.json                                                │
          > {                                                         │
          >   "type": "object",                                       │
          >   "properties": {                                         │
          >     "app-1": { "$ref": "file://../app-1/schema.jsson" }, ─┘
          >     "app-2": { "$ref": "file://../app-2/schema.jsson" }  ─┘
          >   }
          > }
```

Reply:
If I understood correctly, it's not desirable, because, like with the .Files object, Helm is not intended to access a file outside of the current chart. 
This is to avoid malicious behavior

Reply:
I understand. Then, what would be the best way to implement my use case?

Reply:
From my understanding of helm's linting rules, I think your use case is already implemented?  When chart A depends on chart B, chart A's value structure is automatically augmented with a "B" key that contains the value structure for the child B chart (and is linted against B's schema if present).

A more complex case would be that chart B exposes a utility schema (e.g. a schema for a type that's shared between multiple charts' schemas) - this would be similar to a library chart, exposing templates to multiple parent charts.  In theory in this case helm could be extended to allow a schema in a parent chart to import a schema from a child chart (then all the parents can include the same child to share that schema).  That would probably be a fairly simple extension of the groundwork laid in this PR, but I'd prefer to get the basic function in first before getting too complicated.